client/components/data/query-reader-tag/index.jsx

```diff
@@ -3,36 +3,28 @@
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
-import { Component } from 'react';
+import { useEffect } from 'react';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import { requestTags } from 'state/reader/tags/items/actions';
 
-/**
- *  QueryReaderTag takes 1 parameter (the tag ) and will fetch it
- *  and add to the state tree
- */
-class QueryReaderTag extends Component {
-	static propTypes = {
-		requestTag: PropTypes.func.isRequired,
-		tag: PropTypes.string.isRequired,
-	};
-
-	componentDidMount() {
-		this.props.requestTag( this.props.tag );
-	}
+const QueryReaderTag = props => {
+	useEffect(() => {
```

Comment:
there's one subtle thing here: why not simplify this to `useEffect( () => props.requestTag( props.tag ), [ props.tag ] )`?

useEffect expects the return from the effect callback to be a function that cleans up whatever work the callback did. Because of that, we have to use the longer arrow function form to suppress the callback.  

Another option would be `useEffect( () => void props.requestTags( props.tag ), [ props.tag ] );` but that's a pretty odd use of void and not really more concise.